trieste/acquisition/function/active_learning.py

```diff
@@ -323,12 +496,21 @@ def __init__(self, model: ProbabilisticModel, jitter: float):
         :return: The Bayesian Active Learning By Disagreement acquisition function.
         """
         tf.debugging.assert_positive(jitter, message="Jitter must be positive.")
+        assert (
```

Comment:
please use `tf.debugging.Assert` or if condition and then `raise ValueError`

Reply:
also, make sure we have a test for this

Reply:
Sure!